### ISSUE TITLE
why do you choose the calabash, why not use appium ?

### ISSUE THREAD

**Issue by qileilove** [pritibiyani/cross-platform-single-page-example#2]
Well done，thanks for sharing. may be  appium is more easy for crossPlatform testing.

**Comment by pritibiyani** [comment]
@qileilove When we started implementing this calabash was much mature tool than Appium. Perhaps now, that does not hold true. 

I would be more happy, if someone sends pull request with appium as tool. 

And anyway, here point was how we accommodate 3 different platform into one page object rather than tool being used.

**Comment by qileilove** [comment]
@pritibiyani That's true. use one way to implement 3 platform is very helpful, does this framework support parallel testing?  you know, our project have large number of testcases, if we need to run all the cases , which can be extremely time . 
by the way ,our colleagues , a thoughtworker , he has a framework repo about appium parallel test.
[https://github.com/saikrishna321/AppiumTestDistribution](https://github.com/saikrishna321/AppiumTestDistribution)

**Comment by pritibiyani** [comment]
@qileilove  We do parallel testing in our project, but this sample project's intention is rather different. 

And yes, I came across this parallel framework. Thanks for sharing it again.

( And just you know, I'm your colleague as well. )

**Comment by qileilove** [comment]
@pritibiyani really? awsome! I‘m very glad to meet you. I'm good at web automation testing and performance testing, Hope have more communication and learning with everyone in the future.
my email is 'lqi@thoughtworks.com'
